src/consensus/versionbits.cpp

```diff
@@ -0,0 +1,394 @@
+// Copyright (c) 2009-2015 Eric Lombrozo, The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+/*
+                     _             _     _ _                         
+ __   _____ _ __ ___(_) ___  _ __ | |__ (_) |_ ___   ___ _ __  _ __  
```

Comment:
Agree with @jtimon. Also it may be better to split the copyright (c) line into two. One for the initial author and initial year and a second one for the bitcoin core devs.